langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -346,6 +493,15 @@ public isolated function 'map(xml<ItemType> x, @isolatedParam function(ItemType
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml books = xml `<book>Java</book><book>Ballerina</book>`;
+# xml titles = xml ``;
+# 
+# books.forEach(function (xml xmlItem) {
+#     titles += xml `<code>${xmlItem.data()}</code>`;
+# });
```

Comment:
Let's add the output

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/652d5d460c5a0465e3e4659dd0e2bb7002e0317f